rust/hyperlane-base/Cargo.toml

```diff
@@ -21,12 +22,14 @@ ed25519-dalek.workspace = true
 ethers.workspace = true
 eyre.workspace = true
 fuels.workspace = true
+hyper.workspace = true
 futures-util.workspace = true
 itertools.workspace = true
 maplit.workspace = true
 paste.workspace = true
 prometheus.workspace = true
 rocksdb.workspace = true
+reqwest.workspace = true
```

Comment:
I believe this is used just for the metrics endpoint unit test, in which case it should be moved under the `[dev-dependencies]` section below